Mathlib/Algebra/Group/Action/Faithful.lean

```diff
@@ -47,15 +47,20 @@ class FaithfulSMul (M : Type*) (α : Type*) [SMul M α] : Prop where
 export FaithfulSMul (eq_of_smul_eq_smul)
 export FaithfulVAdd (eq_of_vadd_eq_vadd)
 
+instance (R : Type*) [MulOneClass R] : FaithfulSMul R R := ⟨fun {r₁ r₂} h ↦ by simpa using h 1⟩
+
+lemma faithfulVAdd_iff (P : Type*) [VAdd G P] :
+    FaithfulVAdd G P ↔ ∀ {g₁ g₂ : G}, (∀ p : P, g₁ +ᵥ p = g₂ +ᵥ p) → g₁ = g₂ :=
+  ⟨fun h ↦ h.eq_of_vadd_eq_vadd, FaithfulVAdd.mk⟩
+
+lemma faithfulSMul_iff [SMul M α] :
```

Comment:
Can `@[mk_iff]` attribute on `class FaithfulSMul` generate this? Also, please use `@[to_additive]`.

Reply:
Ah thanks. I didn't know about this attribute and now understand your original comment. I found that `@[to_additive (attr := mk_iff)]` generates both theorems